apps/hash-frontend/src/pages/account/workspaces/[shortname]/general.page.tsx

```diff
@@ -0,0 +1,107 @@
+import { EntityPropertiesObject } from "@blockprotocol/graph";
+import { extractBaseUrl } from "@blockprotocol/type-system";
+import { types } from "@local/hash-isomorphic-utils/ontology-types";
+import { Container, Typography } from "@mui/material";
+import { useRouter } from "next/router";
+import { NextSeo } from "next-seo";
+import { useRef } from "react";
+
+import { useBlockProtocolUpdateEntity } from "../../../../components/hooks/block-protocol-functions/knowledge/use-block-protocol-update-entity";
+import {
+  getLayoutWithSidebar,
+  NextPageWithLayout,
+} from "../../../../shared/layout";
+import { useAuthenticatedUser } from "../../../shared/auth-info-context";
+import { OrgForm, OrgFormData } from "../../../shared/org-form";
+
+const OrgGeneralSettingsPage: NextPageWithLayout = () => {
+  const router = useRouter();
+
+  const topRef = useRef<HTMLDivElement>(null);
+
+  const { shortname } = router.query as { shortname: string };
+
+  const { updateEntity } = useBlockProtocolUpdateEntity();
+
+  const { authenticatedUser, refetch } = useAuthenticatedUser();
+
+  const org = authenticatedUser.memberOf.find(
+    (orgOption) => orgOption.shortname === shortname,
+  );
+
+  if (!org) {
+    void router.push("/");
+    return null;
+  }
```

Comment:
Might be nice to have a 404 page instead at some point, maybe we can add a `@todo` for now

Reply:
Done in https://github.com/hashintel/hash/pull/2862/commits/c53b568a046274c2c8d5c794ca492a1e2e08d86e